src/rv32_template.c

```diff
@@ -3,43 +3,75 @@
  * specification version 20191213.
  */
 
+/* Please check files gen-jit-template.py and jit_x64.[ch] to understand custom
+ * domain-specific language of tiered 1 JIT compiler, ecah opcode has
+ * corresonding API. For example, the API of `ld_imm` is `emit_load_imm`.
+ */
+
 /* Internal */
-RVOP(nop, { rv->X[rv_reg_zero] = 0; })
+RVOP(nop, { rv->X[rv_reg_zero] = 0; }, {X64({})})
```

Comment:
How about using shorter form? That is,
```c
RVOP(nop, { rv->X[rv_reg_zero] = 0; }, X64 { } )
```

For interpreter path, we can define a dummy `X64` macro which ignores everything.

Reply:
> How about using shorter form? That is,
> 
> ```c
> RVOP(nop, { rv->X[rv_reg_zero] = 0; }, X64 { } )
> ```
> 
> For interpreter path, we can define a dummy `X64` macro which ignores everything.

It can only be (X64{}), the compiler would print error for X64 {} because the MACRO of RVOP would extend incorrectly.

Reply:
The compiler would print error for X64 {} because the MACRO of RVOP are extended incorrectly.

Reply:
We can still stick to the shorter form `RVOP(nop, { rv->X[rv_reg_zero] = 0; }, X64({ }) )`, right?

Reply:
This form is OK.